lib/bullet.rb

```diff
@@ -167,7 +167,7 @@ def gather_inline_notifications
 
     def perform_out_of_channel_notifications(env = {})
       for_each_active_notifier_with_notification do |notification|
-        notification.url = env['REQUEST_URI']
+        notification.url = env['REQUEST_URI'] || build_request_uri(env)
```

Comment:
move `env['REQUEST_URI'] || build_request_uri(env)` out of for_each_active_notifier_with_notification, so it doesn't need to calculate for each notification.